korgan/professional_consultation_guard.py

```
@@ -0,0 +1,344 @@
+            if unverified:
+                base += "\n\nҚосымша тексеру қажет:\n" + "\n".join(f"• {x}" for x in unverified[:5])
+            return base
+        base = (
+            "Не удалось достаточно подтвердить правовой вывод по официальному источнику. "
+            "Я не буду выдавать непроверенную статью, срок, ставку или подсудность как действующее право."
+        )
+        if unverified:
+            base += "\n\nТребует дополнительной проверки:\n" + "\n".join(f"• {x}" for x in unverified[:5])
+        return base
+
+    law_title = "Расталған құқықтық негіз:" if kk else "Подтверждено по действующему праву РК:"
```

Ruff (0.16.2)

[warning] String contains ambiguous `Р` (CYRILLIC CAPITAL LETTER ER). Did you mean `P` (LATIN CAPITAL LETTER P)?

(RUF001)